Assets/Scripts/SpawnManager.cs

```diff
@@ -19,6 +19,8 @@ void Start()
 				SpawnPoints = GameObject.FindGameObjectsWithTag("Respawn");
 	            Vector3 spawnPositon = GetRandomSpawnPoint();
 	            LocalPlayer = PhotonNetwork.Instantiate("NetPlayer", spawnPositon, Quaternion.identity, 0);
+                // todo: Delete once event system is in place
+                LocalPlayer.GetComponent<LifeManager>().mDepositManager = GameObject.Find("DepositBox").GetComponent<DepositManager>();

```

Comment:
The life manager right now has a deposit manager reference so that it can alert the deposit managers when the character dies. This will be replaced with our event system